client/cli/src/commands/inspect_key.rs

```diff
@@ -28,7 +28,7 @@ use structopt::StructOpt;
 	name = "inspect-key",
 	about = "Gets a public key and a SS58 address from the provided Secret URI"
 )]
-pub struct InspectKeyCmd {
+pub struct InspectCmd {
```

Comment:
Why is that renamed? please rename it back to the old name.

Reply:
It is changed to match the name of the command itself, I don't _think_ this is required for the CLI package we are using, is it? 
if not, of course, I can change back

Reply:
It is not required to match.

Reply:
reverted